src/VolumeDrawable.ts

```diff
@@ -662,62 +667,28 @@ export default class VolumeDrawable {
 
     // create new
     if (isPathtrace) {
-      this.pathTracedVolume = new PathTracedVolume(this.volume);
+      this.pathTracedVolume = new PathTracedVolume(this.volume, this.settings);
```

Comment:
not sure if we still need to keep this.pathTracedVolume and this.rayMarchedAtlasVolume as member variables anymore?

Reply:
Mm, yeah, I can make that change. I can also remove the PT boolean flag and we can just check for what the current VolumeRenderImpl is an instance of.